src/components/radiobutton/radiobutton.component.ts

```diff
@@ -126,4 +138,9 @@ export class RadioButtonComponent<T = any> implements ControlValueAccessor {
         this.disabled = isDisabled;
         this._changeDetector.markForCheck();
     }
+
+    setTabIndex(tabIndex): void {
```

Comment:
```suggestion
    setInternalTabindex(tabIndex): void {
```
Or something else, since this component now has two tabindex properties. Also, add descriptions to your functions.